src/corppa/utils/generate_page_set.py

```diff
@@ -0,0 +1,127 @@
+"""
+Utility for generating a PPA page set.
+
+This method takes three inputs: (1) an input csv, (2) an output csv, and
+(3) the size of the page set.
+
+The input CSV file must have the following fields:
+    * work_id: PPA work id
+    * page_start: Starting index for page range being considered for this work
+    * page_end: Ending index for page range being considered for this work
+    * poery_pages: Comma separated list of page numbers containing poetry
```

Comment:
I don't know if my comment belongs here or on the google doc you shared with your steps for constructing the dataset, but how did you construct the input CSV?

Reply:
I'll make a comment in the google doc, but it was a relatively simple generation from the poem-focused test set spreadsheet.